server/routes/stripe.js

```diff
@@ -103,4 +104,28 @@ router.post('/webhook', bodyParser.raw({ type: 'application/json' }), async (req
   res.json({ received: true });
 });
 
+// called from frontend after stripe payment confirmation
+// redirect to a thank you page
+router.get('/payment/success/:id', redirectLogin, async (req, res) => {
+  try {
+    const wishCardId  = req.params.id;
+    const wishCard = await WishCardRepository.getWishCardByObjectId(mongoSanitize.sanitize(wishCardId));
+    const totalItemPrice = await calculateWishItemTotalPrice(wishCard.wishItemPrice);
```

Comment:
this will change since the itemprice doesnt reflect the price if a user added money for us to the donation,

you might have to get that from the donationsmodel

Reply:
I cant do that, same problem as above